.github/workflows/nroff-elves.sh

```diff
@@ -70,6 +70,10 @@ git push --set-upstream origin $branch_name
 url=`gh pr create --base $BASE_REF --title 'Update nroff-generated man pages' --body ''`
 pr_num=`echo $url | cut -d/ -f7`
 
+# skip the remaining steps for now since merge now requires
+# appproval which cannot be done automatically.
+exit 0
```

Comment:
Is there a benefit in exiting instead of just removing the below code?

Reply:
keep those code for future reference in case the repo policy is changed again or a new way of merging become available.